src/components/Forms/Volunteer.jsx

```diff
@@ -18,6 +19,48 @@ const Volunteer = () => {
   const [requirements, setRequirements] = useState(Requirements);
 
   const handleSubmit = () => {
+    if (volunteer.first === "") {
+      toast("❌ Please include your first name!");
+      return;
+    }
+    if (volunteer.last === "") {
+      toast("❌ Please include last name!");
+      return;
+    }
+    if (volunteer.email == "") {
+      toast("❌ Please include your email!");
+      return;
+    }
+    if (volunteer.phone == "") {
+      toast("❌ Please include your phone number!");
+      return;
+    }
+    if (volunteer.major == "") {
+      toast("❌ Please include major!");
+      return;
+    }
+    if (volunteer.availability == null) {
+      toast("❌ Please select your availability!");
+      return;
+    }
+    if (volunteer.shirt == "") {
+      toast("❌ Please include shirt size!");
+      return;
+    }
+    if (volunteer.grade == "") {
+      toast("❌ Please complete all fields!");
+      return;
+    }
+    if (requirements.inPerson.state == false) {
```

Comment:
u should be able to do a `some` on the array, since the number of checkboxes may increase or decrease